pkg/services/live/pipeline/json_to_frame.go

```diff
@@ -108,6 +108,7 @@ func jsonDocToFrame(name string, body []byte, fields map[string]Field, nowTimeFu
 	}
 
 	f := data.NewFieldFromFieldType(data.FieldTypeTime, 1)
+	f.Name = "Time"
```

Comment:
Fixed it here rather than in frontend as `name` is marked as non-nullable https://github.com/grafana/grafana/blob/c41b5d80a3bdceb658f3815e36671155000eeef3/packages/grafana-data/src/dataframe/DataFrameJSON.ts#L89 so we always expect it to be filled